resources/dev_install_conda.sh

```
@@ -0,0 +1,20 @@
+#! /usr/bin/env bash
+
+conda env create --prefix harpy/.conda/harpy $1 --file harpy/resources/harpy.yaml
+
+conda activate harpy/.conda/harpy
+
+mkdir -p ${CONDA_PREFIX}/bin
+
+cd harpy
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)